### DISCUSSION TITLE
Finding ranking results

### DISCUSSION THREAD

**Discussion by gephiuserumpteen** [gephi/gephi#2816] [Q&A]
Hi - I am new to Gephi so apologies if this is a basic question. I am using Gephi 0.10.1 but the tutorials I've found seem to be around Gephi 0.8 and there are a couple of points I'm having difficulty transferring over. Currently I am stuck on ranking. 

I would like to see the ranking result table, as on slide 11 of this guide https://gephi.org/tutorials/gephi-tutorial-quick_start.pdf

However, the thing I think I have to click in the appearance tab is greyed out and I'm not sure why. (See image below.)

![image](https://github.com/gephi/gephi/assets/134067485/7b88987c-51f1-4c66-a4f3-e2d0d0395c0b)

Thanks in advance for any help :)

**Comment by mbastian** [comment]
Hi @gephiuserumpteen, the ranking table as shown on the quick start is no longer there. The quick start is outdated. You can however get the same data in the Data Laboratory and export it.

**Reply by gephiuserumpteen** [reply]
Thank you!
